crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -352,37 +370,57 @@ impl RoomEventCacheInner {
     }
 
     #[instrument(skip_all, fields(room_id = %self.room_id))]
-    pub(super) async fn handle_joined_room_update(&self, updates: JoinedRoomUpdate) -> Result<()> {
-        self.handle_timeline(
-            updates.timeline,
-            updates.ephemeral.clone(),
-            updates.ambiguity_changes,
-        )
-        .await?;
+    pub(super) async fn handle_joined_room_update(
+        &self,
+        updates: JoinedRoomUpdate,
+        generic_room_event_cache_update_sender: &Sender<GenericRoomEventCacheUpdate>,
+    ) -> Result<()> {
+        if self
+            .handle_timeline(updates.timeline, updates.ephemeral.clone(), updates.ambiguity_changes)
+            .await?
+        {
+            let _ = generic_room_event_cache_update_sender.send(
```

Comment:
So we ignore errors from `send`? Why? Should we log them at least?

Reply:
[Quoting the doc](https://docs.rs/tokio/latest/tokio/sync/broadcast/error/struct.SendError.html):

> A send operation can only fail if there are no active receivers, implying that the message could never be received. The error contains the message being sent as a payload so it can be recovered.

We don't care if there is no more receivers, it's _fine_ to ignore the error in this case.